cinema/tests/test_movie_api.py

```diff
@@ -157,3 +154,12 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
It seems like you're checking if the key 'movie_image' exists in the response data. It would be more accurate to also check if the value of 'movie_image' is the expected image url.